models/campaign.go

```diff
@@ -157,6 +158,26 @@ func (c *Campaign) UpdateStatus(s string) error {
 func (c *Campaign) AddEvent(e *Event) error {
 	e.CampaignId = c.Id
 	e.Time = time.Now().UTC()
+
+  //INFO send webhook
+  pl := map[string]interface{} {
```

Comment:
Instead of making a new map, I’d still strongly recommend just sending the event itself to SendAll. That way, the event sent to the webhook is the same one returned by the API and stored in the DB.

Reply:
Also note that since we’re putting the call here, you won’t need to call the webhook anywhere else - all events are meant to go through here. :)

Reply:
* sending an event - done

* can you elaborate on "you won’t need to call the webhook anywhere else - all events are meant to go through here. :)" ?